forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -3207,61 +3184,28 @@ public void resolve() {
             final String[] k = keyword.split(":");
             final String manacost = k[1];
 
-            String effect = "AB$ RepeatEach | Cost$ " + manacost + " ExileFromGrave<1/CARDNAME> " +
-                    "| ActivationZone$ Graveyard | ClearRememberedBeforeLoop$ True | RepeatPlayers$ Opponent" +
-                    "| PrecostDesc$ Encore | CostDesc$ " + ManaCostParser.parse(manacost) +
+            final String effect = "AB$ CopyPermanent | Cost$ " + manacost + " ExileFromGrave<1/CARDNAME> | ActivationZone$ Graveyard" +
+                    "| Defined$ Self | PumpKeywords$ Haste | RememberTokens$ True | ForEach$ Opponent | CleanupForEach$ EOT" +
+                    "| AtEOT$ Sacrifice | PrecostDesc$ Encore | CostDesc$ " + ManaCostParser.parse(manacost) +
                     "| SpellDescription$ (" + inst.getReminderText() + ")";
 
-            final String copyStr = "DB$ CopyPermanent | Defined$ Self | ImprintTokens$ True " +
-                    "| AddKeywords$ Haste | RememberTokens$ True | TokenRemembered$ Player.IsRemembered " +
-                    "| AddStaticAbilities$ MustAttack";
-
-            final String pumpStr = "DB$ Animate | Defined$ Creature.IsRemembered | staticAbilities$ AttackChosen ";
-
+            final SpellAbility sa = AbilityFactory.getAbility(effect, card);
+            final String animateStr = "DB$ Animate | Defined$ Remembered | staticAbilities$ AttackChosen";
+            final AbilitySub animateSub = (AbilitySub) AbilityFactory.getAbility(animateStr, card);
```

Comment:
from the Rules for Encore this probably should be an Extra Effect, not an Animate that can be removed by "lose abilities"

Reply:
fixed